src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+    protected String at;
+
+    public Event (String taskName, String at) {
+        super(taskName);
+        this.at = at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + "(at: " +
```

Comment:
For this case, it might be better to use String. format to improve on the code readability.